awx/api/generics.py

```diff
@@ -217,7 +217,7 @@ def finalize_response(self, request, response, *args, **kwargs):
             if hasattr(self, '__init_request_error__'):
                 response = self.handle_exception(self.__init_request_error__)
             if response.status_code == 401:
-                response.data['detail'] += ' To establish a login session, visit /api/login/.'
+                response.data['detail'] += _(' To establish a login session, visit') + '/api/login/.'
```

Comment:
A space is needed between the translated string and the api path.

Reply:
Nice catch.  I just fixed this.